README.md

```diff
@@ -6,6 +6,8 @@
 
 `harmony-py` is a work-in-progress, is not feature complete, and should only be used if you would like to test `harmony-py` functionality.
 
+![Python package](https://github.com/nasa/harmony-py/workflows/Python%20package/badge.svg)
```

Comment:
Just curious - what does this badge show?

Reply:
I'm not sure (I didn't put it there).

Reply:
![image](https://user-images.githubusercontent.com/14251936/108745436-4b5c8c80-74f8-11eb-9a09-b84c1b220a00.png)